wizard/core/state.py

```diff
@@ -0,0 +1,144 @@
+"""Streamlit session state.
```

Comment:
The present file seems Streamlit-related. In `wizard/core`, the goal was to only put re-usable generic Python functions: like infering types, handling files, etc.

Is it possible to have those files isolated somewhere? You can certainly rename the folder, but I'd like to have the business logic isolated from the Streamlit logic (e.g., in case, we have to refactor with another web framework).

Reply:
moved to utils instead, it's state related, but I did notice that core was streamlit free.